packages/server/utils/OpenAIServerManager.ts

```diff
@@ -35,7 +36,19 @@ class OpenAIServerManager {
       })
       return (response.data.choices[0]?.text?.trim() as string) ?? null
     } catch (e) {
-      const error = e instanceof Error ? e : new Error('OpenAI failed to getSummary')
+      if (e instanceof Error) {
+        const rateLimitStatusCode = 429
+        if (
+          (e as AxiosError).response?.status === rateLimitStatusCode &&
+          (e as AxiosError).response?.data?.error
+        ) {
+          sendToSentry((e as AxiosError).response?.data?.error)
```

Comment:
-1 I assume `.error` is a string message and should be wrapped in an error, right?